EPRN.Portal/Controllers/PrnsController.cs

```diff
@@ -35,9 +35,15 @@ public async Task<IActionResult> PrnSavedAsDraftConfirmation(int? id)
             return View(viewModel);
         }
 
+        [HttpGet]
         public async Task<IActionResult> Action(int? id)
         {
-            var viewModel = await _prnService.ActionPrnViewModel();
+            if (id == null)
+            {
```

Comment:
Don't need the braces if onles one line after the if

Reply:
Done.